apps/marketplace/components/Opportunities/OpportunitiesFilters.js

```diff
@@ -62,17 +63,33 @@ export class OpportunitiesFiltersComponent extends BaseForm {
   changeAccordion(type, isOpen) {
     switch (type) {
       case 'status':
-        this.setState({ statusAccordionOpen: isOpen })
+        this.setState({
+          statusAccordionOpen: isOpen,
+          locationAccordionOpen: false,
+          typeAccordionOpen: false
+        })
         break
       case 'location':
-        this.setState({ locationAccordionOpen: isOpen })
+        this.setState({
+          locationAccordionOpen: isOpen,
+          statusAccordionOpen: false,
+          typeAccordionOpen: false
+        })
+        break
+      case 'type':
+        this.setState({
+          typeAccordionOpen: isOpen,
+          locationAccordionOpen: false,
+          statusAccordionOpen: false
+        })
```

Comment:
Can we reduce some repetition in this `switch` if we did something like this:
```
case 'location':
case 'status':
case 'type:
  this.setState({
      locationAccordionOpen: type == 'location' ? isOpen : false,
      statusAccordionOpen: type == 'status' ? isOpen : false,
      typeAccordionOpen: type == 'type' ? isOpen : false
  })
```

Reply:
done